python/ray/includes/task.pxi

```diff
@@ -229,6 +231,13 @@ cdef class TaskSpec:
         return ObjectID(
             self.task_spec.get().ActorCreationDummyObjectId().Binary())
 
+    def previous_actor_task_dummy_object_id(self):
+        """Return the object ID of the previously executed actor task."""
+        if not self.is_actor_task():
+            return ObjectID.nil()
+        return ObjectID(
```

Comment:
Nit, prefer returning in an `else` block here.

Reply:
I'm strongly against returning in an `else` block in these cases, but looks like the usage is mixed in the Python code from a quick glance. We should pick one and stick to it eventually. For now, will leave this as-is to keep consistent with the other near-identical functions.

Reply:
Returning in `else` is more readable to me, but I don't feel strongly about it.